svc/GridExportService.ts

```diff
@@ -141,6 +145,14 @@ export class GridExportService extends HoistService {
             download(blob, `${filename}${fileExt}`, contentType);
             await dismissStartToast();
             XH.successToast('Export complete.');
+
+            if (track) {
+                XH.track({
+                    category: 'Navigation',
+                    message: `Downloaded ${filename}${fileExt}`,
+                    data: {rows: rows.length, columns: exportColumns.length}
+                });
+            }
```

Comment:
The basic message relies on a helpful filename to know which grid was exported/where it can be found. Do you think it's helpful enough? It might encourage providing specific filenames instead of using the default "export."

Reply:
Looks good.  Not 100% sure about the category.  It's not quite "Navigation", but am unsure what, if any, new category of message we might introduce.  "Data"?  "Export"?  "Grids?"